gameplay/avalonRoom.js

```diff
@@ -1629,14 +1643,16 @@ module.exports = function (host_, roomId_, io_, maxNumPlayers_, newRoomPassword_
 			if (this.host === socket.request.user.username) {
 				
 				this.socketsOfPlayers.splice(getIndexFromUsername(this.socketsOfPlayers, username), 1);
-				
-				console.log("Kicked player: " + username);
-
-				this.sendText(this.socketsOfPlayers, "Player " + username + " has been kicked by the host.", "server-text");
-
 				// Ban them from this room
-				this.kickedPlayers[username] = true;
-
+				// Add ban time start
+				this.kickedPlayers[username] = {
+					"status": true,
+					"startTime": (new Date).getTime()
```

Comment:
I would probably leave it as a Date object instead of converting to millis. It works fine as it is, but if in the future you still wanted the Date object functions then you'd have to recreate a new Date object lol. No need to change unless you really want to though!

Reply:
Noted. Was also about ban times.

Accepted. Will be addressed in the next commit.